examples/microphone_streaming_example.py

```diff
@@ -83,6 +81,7 @@ def generator(self):
 
             yield b''.join(data)
 
+
```

Comment:
Did the linter tell you to add this line?

Reply:
Yep.